test/api_stability/3.0.3/0007-NOAPI-INCLUDE-MISC-split-type_-pack-list-pack-algori.patch

```diff
@@ -0,0 +1,260 @@
+From 08673ed41ee153133f0578039862f5cad5a11dcd Mon Sep 17 00:00:00 2001
+From: marehr <marehr-github@marehr.dialup.fu-berlin.de>
+Date: Fri, 3 Sep 2021 12:20:18 +0200
+Subject: [PATCH 7/7] [NOAPI] [INCLUDE] [MISC] split type_(pack|list) pack
```

Comment:
Should this file be here?

Reply:
Yes, as it fixes the API stability test. (See if old previous seqan3 version still compiles)